cmd/co/diff.go

```diff
@@ -14,7 +14,7 @@ import (
 	"strings"
 
 	"github.com/fatih/color"
-	"github.com/lukealbao/co"
+	codeowners "github.com/lukealbao/co"
```

Comment:
vscode insists on doing this whenever I save a file. I don't know how to make it stop, so I've given in and committed this.